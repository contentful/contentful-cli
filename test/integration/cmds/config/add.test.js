const nixt = require('nixt')
const { join } = require('path')

const bin = join(__dirname, './../../../../', 'bin')

const app = () => {
  return nixt({ newlines: true })
    .cwd(bin)
    .base('./contentful.js ')
    .clone()
}

test('config add throws error when option as is empty', done => {
  app()
    .run('config add --as')
    .code(1)
    .expect(result => {
      const resultText = result.stderr.trim()
      expect(resultText).toMatchSnapshot('Not enough arguments following: as')
    })
    .end(done)
})

test('config add throws error when option mt is empty', done => {
  app()
    .run('config add --mt')
    .code(1)
    .expect(result => {
      const resultText = result.stderr.trim()
      expect(resultText).toMatchSnapshot('Not enough arguments following: mt')
    })
    .end(done)
})

test('config add throws error when option ae is empty', done => {
  app()
    .run('config add --ae')
    .code(1)
    .expect(result => {
      const resultText = result.stderr.trim()
      expect(resultText).toMatchSnapshot('Not enough arguments following: ae')
    })
    .end(done)
})
