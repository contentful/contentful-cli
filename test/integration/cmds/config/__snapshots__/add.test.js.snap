// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`config add throws error when option ae is empty: Not enough arguments following: ae 1`] = `"Not enough arguments following: ae"`;

exports[`config add throws error when option as is empty: Not enough arguments following: as 1`] = `"Not enough arguments following: as"`;

exports[`config add throws error when option mt is empty: Not enough arguments following: mt 1`] = `"Not enough arguments following: mt"`;
